### ISSUE TITLE
SOMA collection : to_seurat() not bringing back all assays

### ISSUE THREAD

**Issue by dan11mcguire** [TileDB-Inc/tiledbsc#82]
If the first assay is not named "RNA", there will be an assay missing when suign "to_seurat" to bring seurat object into memory.

The fix in this case I think could be here:

![image](https://user-images.githubusercontent.com/49620432/199785122-5a18ff23-7a83-498e-ac20-309cb8e70fdc.png)

Example:
```
`library(tiledbsc)
library(SeuratObject)

soco_uri <- file.path(tempdir(), "soco-pbmc3k")
soco <- SOMACollection$new(uri = soco_uri)
pbmc3k <- dataset_seurat_pbmc3k()
### add some more assays
rna2 <-
CreateAssayObject(counts = pbmc3k[["RNA"]]@counts
                  ,assay = "rna2")

rna3 <-
CreateAssayObject(counts = pbmc3k[["RNA"]]@counts
                  ,assay = "rna3")

pbmc3k[["rna2"]] <- rna2
pbmc3k[["rna3"]] <- rna3

new_seurat <- Seurat::CreateSeuratObject(counts = pbmc3k[["rna2"]]@counts
                                         ,meta.data = pbmc3k@meta.data
                                         ,assay = "rna2")

new_seurat[["rna3"]] <- rna3
new_seurat[["RNA"]] <- pbmc3k[["RNA"]]
soco$from_seurat(new_seurat)

### "RNA" is not the first soma, which will cause the error
soco$somas

bringback <- soco$to_seurat()
Seurat::Assays(bringback)`
```

![image](https://user-images.githubusercontent.com/49620432/199784123-71488fc9-6259-4025-9a52-88652eb21c2f.png)

**Comment by aaronwolen** [comment]
@dan11mcguire great catch and your proposed fix looks perfect! Thanks very much, I'll get this patched ASAP.

**Comment by aaronwolen** [comment]
Closed in #83
